app/jobs/delete_resources_job.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Load JSON resources from a url, persist them, and index them
+class DeleteResourcesJob < ApplicationJob
+  queue_as :default
+
+  def perform(url, exhibit)
+    resp = Faraday.get(url)
+
+    resource = resp.body.split("\n")
+    resource.each_with_index do |item, index|
+      remove_resource(item, exhibit, index)
+    end
+    logger.info("#{resource.count} records were created from #{url}.")
```

Comment:
Should this be "...records were removed from..."?

Reply:
Yes! Thank you Aaron. That's fixed.